pkgs/by-name/cr/crusader/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  autoPatchelfHook,
+  lib,
+  libgcc,
+  libGL,
+  fetchFromGitHub,
+  fontconfig,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+  libxkbcommon,
+  xorg,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "crusader";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "Zoxc";
+    repo = "crusader";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-M5zMOOYDS91p0EuDSlQ3K6eiVQpbX6953q+cXBMix2s=";
+  };
+
+  sourceRoot = "${src.name}/src";
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
```

Comment:
IIRC there is an argument that you can only specify a hash of the cargo deps, I don't remember the name tho

Reply:
Isn't that what I'm doing by specifying `hash` to `rustPlatform.fetchCargoVendor`?

Reply:
Yeah, but there is an argument where you just pass the hash and it becomes equivalent to this.

Reply:
Nevermind, I changed it to the same style I see almost everywhere else:

```
  useFetchCargoVendor = true;
  cargoHash = "sha256-f0TWiRX203/gNsa9UEr/1Bv+kUxLAK/Zlw+S693xZlE=";
```